### DISCUSSION TITLE
Error when enter credentials

### DISCUSSION THREAD

**Discussion by WernerP63** [DarwinsBuddy/WienerNetzeSmartmeter#249] [Q&A] [answered]
First of all, I like your integration - but since August 2024 I dont get data and when I try to reconnect there a missing key reported :(

custom_components.wnsm.api.errors.SmartmeterConnectionError: Could not obtain API Key - no match
2024-08-05 15:04:51.429 ERROR (MainThread) [custom_components.wnsm.config_flow] Error validating Wiener Netze auth
2024-08-05 15:04:51.429 ERROR (MainThread) [custom_components.wnsm.config_flow] Could not obtain API Key - no match

**Comment by WernerP63** [comment, marked as answer]
Sorry - didn´t see may fix https://github.com/DarwinsBuddy/WienerNetzeSmartmeter/issues/246 and https://github.com/DarwinsBuddy/WienerNetzeSmartmeter/issues/247

**Comment by DarwinsBuddy** [comment]
closing. happy to see this resolved
